tests/abort1.rs

```diff
@@ -97,7 +97,9 @@ fn test_stop_taker_after_setup() {
                 .balance_incoming_swap_coins(Some(&all_utxos))
                 .unwrap();
 
-            let live_contract_balance = wallet.balance_live_contract(Some(&all_utxos)).unwrap();
+            let live_contract_balance = wallet
+                .balance_live_timelock_contract(Some(&all_utxos))
+                .unwrap();
```

Comment:
We now don't need to fetch individual balances. We can call `get_balances` and assert values in the balance structure. This will simplify the code, as well as put `get_balances` in the coverage.

Same changes will be applicable to all tests and mod.rs also. This can be done as a followup PR.